tests/e2e/standalone/init_test.go

```diff
@@ -128,6 +128,69 @@ func TestStandaloneInit(t *testing.T) {
 		verifyConfigs(t, daprPath)
 	})
 
+	t.Run("init with --dapr-path flag", func(t *testing.T) {
+		// Ensure a clean environment
+		must(t, cmdUninstall, "failed to uninstall Dapr")
+
+		daprPath, err := os.MkdirTemp("", "dapr-e2e-init-with-flag-*")
+		assert.NoError(t, err)
+		defer os.RemoveAll(daprPath) // clean up
+
+		output, err := cmdInit(daprRuntimeVersion, "--dapr-path", daprPath)
```

Comment:
Looks like I don't have permissions to edit this PR, but you should fix this one.

`cmdInit` takes an `args...` variable that is used with the `init` command. This should be changed to the following across all occurrences. @mikeb26 

```go
		output, err := cmdInit("--runtime-version", daprRuntimeVersion, "--dapr-path", daprPath)
```